dmutils/email.py

```diff
@@ -231,3 +232,95 @@ def decode_invitation_token(encoded_token):
 
 def token_created_before_password_last_changed(token_timestamp, user_timestamp):
     return token_timestamp < user_timestamp
+
+
+class DMNotifyClient(object):
```

Comment:
@allait I'd kind of prefer for this to have a separate `notify.py` file. What do you think?